gsadjust/pyqt_models.py

```diff
@@ -1537,16 +1562,9 @@ def headerData(self, section, orientation, role=QtCore.Qt.DisplayRole):
             if orientation == QtCore.Qt.Horizontal:
                 return QVariant(int(QtCore.Qt.AlignLeft | QtCore.Qt.AlignVCenter))
             return QVariant(int(QtCore.Qt.AlignRight | QtCore.Qt.AlignVCenter))
-        if role != QtCore.Qt.DisplayRole:
-            return QVariant()
-        if orientation == QtCore.Qt.Horizontal:
-            if section == 0:
-                return QVariant("From")
-            elif section == 1:
-                return QVariant("To")
-            elif section == 2:
-                return QVariant("Delta g")
-        return QVariant(int(section + 1))
+
+        if role == QtCore.Qt.DisplayRole and orientation == QtCore.Qt.Horizontal:
+            return self._headers.get(section, section + 1)
```

Comment:
All models now have this construct for getting the headers for display. We define a `_headers` `dict` on the class which maps from the column constants to the given header. Here we get the column using `section` (an int) or return `section + 1` if that column is not defined.